src/main/java/me/eccentric_nz/tardisvortexmanipulator/TvmRecipe.java

```diff
@@ -53,11 +54,12 @@ public ShapedRecipe makeRecipe() {
             itemStack = new ItemStack(material, amount);
         }
         ItemMeta itemMeta = itemStack.getItemMeta();
-        itemMeta.setDisplayName("Vortex Manipulator");
+        itemMeta.setDisplayName(ChatColor.RESET + "Vortex Manipulator");
```

Comment:
Why? This is implicit anyway.

Reply:
The italics were ugly. I think the same with every TARDIS item, but I am not going to change those. However, I _can_ do it for the GUI icons, with no side effects.